waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -91,22 +97,31 @@ method getAll*(db: WakuMessageStore, onData: message_store.DataProc): MessageSto
   proc msg(s: ptr sqlite3_stmt) {.raises: [Defect, Exception].} =
     gotMessages = true
     let
-      timestamp = sqlite3_column_int64(s, 0)
+      receiverTimestamp = sqlite3_column_double(s, 0)
+
       topic = cast[ptr UncheckedArray[byte]](sqlite3_column_blob(s, 1))
       topicL = sqlite3_column_bytes(s,1)
```

Comment:
Is this topic length? I'd call it `topicLen` or something like this. Similar with `p` and `l` - not very descriptive, have a hard time following

Reply:
Sure, done!